server/log_rpc_server_test.go

```diff
@@ -25,6 +25,8 @@ import (
 
 	"github.com/golang/mock/gomock"
 	"github.com/golang/protobuf/proto"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
```

Comment:
This is a new import that is different from the 54 existing places that use `pretty.Compare`; is it possible to be consistent or is that too much of a pain?

Reply:
@RJPercival also wants this for #1531, so this will be consistent with that :-)